app/domain/authentication/authn_jwt/jwt_initial_input_validation.rb

```diff
@@ -0,0 +1,40 @@
+require 'command_class'
+
+# This class
+module Authentication
+  module AuthnJwt
+
+    JWTInitialInputValidation ||= CommandClass.new(
+      dependencies: {
+        enabled_authenticators: InstalledAuthenticators.enabled_authenticators_str(ENV),
+        # ValidateWebserviceIsWhitelisted calls ValidateAccountExists
+        validate_webservice_is_whitelisted: Security::ValidateWebserviceIsWhitelisted.new,
+      },
+      inputs: %i[authentication_parameters]
+    ) do
+
+      def call
+        validate_webservice_is_whitelisted
+      end
```

Comment:
what about `validate_authenticator_exists`?

Reply:
also - validations on the user will go in another class, correct?

Reply:
1. Following my discussion with @nessiLahav `validate_authenticator_exists` is a `ConfigurationFactory` responsibility. It will return a general or concrete vendor authn implementation class instance and will throw an exception if `vendor` does not exist.
2. Validation of the user is possible only after token signature validation. It's in another class, right.